frontend/packages/console-app/src/actions/providers/web-terminal-provider.ts

```diff
@@ -0,0 +1,7 @@
+import { FLAG_WEB_TERMINAL } from '@console/dev-console/src/const';
```

Comment:
This adds a dependency from console-app to dev-console, aka from the admin perspective to the developer perspective. We should avoid this direction.

The web terminal / cloud shell feature is almost completely implemented in the console-app and console-shared packages. We should do it similarly for the new customization.

Please move all related file into the `packages/console-app/src/components/cloud-shell/` folder. If you prefer into a customization subfolder.